apps/remote_control/lib/lexical/remote_control/project_node.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Lexical.RemoteControl.ProjectNode do
+  use GenServer
+
+  def start_link(options) do
+    GenServer.start_link(__MODULE__, options, [])
+  end
+
+  def init(options) do
+    wait_boot = options[:wait_boot] || 5_000
+    send(self(), :start_remote_control)
```

Comment:
use `handle_continue` here:
```elixir
{:ok, state, {:continue, :start_remote_control}}
end

# then 
def handle_continue(:start_remote_control, state) do 
  ...
end
```

This genserver should probably go into a dynamic supervisor, and it should do the monitoring of the remote node, right?

Then you can do something like: 
```elixir
ProjectNode.wait_until_started(project) 
```

Reply:
I think you mean: df3293e50fc940a92274e1055ab2b3a8883afdae

However, I still don't know how to halt the project node quickly when the server node halts.